app/js/components/Reputation.scss

```diff
@@ -0,0 +1,33 @@
+.rating-container {
```

Comment:
What about keeping the container name the same as the component:
e.g => reputation (or feel free to change the name of the component)